tests/suites/test_suite_ecp.function

```diff
@@ -674,6 +674,52 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE */
+void ecp_test_mul( int id, char * n_hex,
+                   char *  Px_hex, char *  Py_hex, char *  Pz_hex,
+                   char * nPx_hex, char * nPy_hex, char * nPz_hex,
+                   int res )
```

Comment:
Consider using `expected_ret` instead of `res`, like in `tests/suites/test_suite_ecdh.function`